README.md

```diff
@@ -0,0 +1,24 @@
+# Laserfiche API Client Core Maven
+Maven build implementation of various foundational APIs for Laserfiche, including authorization APIs such as OAuth 2.0 flows for secure and easy access to Laserfiche APIs.

```

Comment:
I am not sure "Maven build implementation" means. But this could just be my problems.

Reply:
removed maven build